lib/node_types/esp/src/device.h

```diff
@@ -77,19 +194,60 @@ class Callback_Link {
         }
 };
 
+/**
+ * @class Subdevice
+ * @brief Represents one value/channel of a Device
+ * 
+ * PURPOSE
+ * =======
+ * Many physical devices produce multiple values. For example:
+ * - DHT sensor: temperature AND humidity
+ * - RGB LED: red, green, blue channels
+ * - GPS: latitude, longitude, altitude
```

Comment:
we don't have gps (yet), maybe take an imu like the mpu6050 as example that creates different axis values and accelerations - sends three comma separated values on ypr topic and also three on acc topic